Lib/fontTools/varLib/__init__.py

```diff
@@ -917,7 +918,11 @@ def build(designspace, master_finder=lambda s:s, exclude=[], optimize=True):
 	if 'cvar' not in exclude and 'glyf' in vf:
 		_merge_TTHinting(vf, model, master_fonts)
 	if 'GSUB' not in exclude and ds.rules:
-		_add_GSUB_feature_variations(vf, ds.axes, ds.internal_axis_supports, ds.rules, ds.rulesProcessingLast)
+		if ds.rulesProcessingLast:
```

Comment:
how about the `processing="last"` is implicitly set whenever the lib key is present? It can't be otherwise, can it?

Reply:
I mean, if one uses the lib key to set the custom tag, then `processing` attribute is irrelevant, since what it really matters is whether it is "rvrn" (which will be processed first) or any other feature tag (which will be processed last or following normal lookup order)

Reply:
Yeah, thanks, that makes sense. I'll change it.